types/mobx-apollo/package.json

```diff
@@ -0,0 +1,8 @@
+{
+    "private": true,
+    "dependencies": {
+        "apollo-client": "^2.0.0",
+        "graphql-tag": "^2.0.0",
```

Comment:
This is actually just a dev dependency, but the linter did not allow that -- are these actually published as the pacakge depdencies?